src/vmm/src/devices/virtio/vsock/csm/connection.rs

```diff
@@ -93,7 +93,8 @@ use super::super::packet::VsockPacket;
 use super::super::{VsockChannel, VsockEpollListener, VsockError};
 use super::txbuf::TxBuf;
 use super::{defs, ConnState, PendingRx, PendingRxSet, VsockCsmError};
-use crate::logger::{IncMetric, METRICS};
+use crate::devices::virtio::vsock::vsock_metrics::VSOCK_METRICS;
```

Comment:
nit: commit message:
```diff
-refactor(metrics): Use VSOCK_METRICS
+refactor(metrics): Use VSOCK_METRICS instead of METRICS.vsock
```
Also, maybe could add a sentence on why it is better?

Reply:
done. Thanks!